Resources/views/standard_layout.html.twig

```diff
@@ -135,7 +135,7 @@ file that was distributed with this source code.
                                 <img src="{{ asset(sonata_admin.adminPool.titlelogo) }}" alt="{{ sonata_admin.adminPool.title }}">
                             {% endif %}
                             {% if 'single_text' == sonata_admin.adminPool.getOption('title_mode') or 'both' == sonata_admin.adminPool.getOption('title_mode') %}
-                                <span>{{ sonata_admin.adminPool.title }}</span>
+                                <span>{{ sonata_admin.adminPool.title|trans({}, 'messages') }}</span>
```

Comment:
but our default translation domain is `SonataAdminBundle`, or shall we use `messages` because its translation is based on "userland/project stuff" ?